src/ImageService.Models.Dto/Responses/ImageDataResponse .cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+
+namespace LT.DigitalOffice.ImageService.Models.Dto.Responses
+{
+    public class ImageDataResponse
```

Comment:
record, all models in dto project are records or enum

Reply:
done